airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/ReplicationActivityImpl.java

```diff
@@ -116,7 +117,8 @@ public ReplicationActivityImpl(@Named("containerOrchestratorConfig") final Optio
   public StandardSyncOutput replicate(final JobRunConfig jobRunConfig,
                                       final IntegrationLauncherConfig sourceLauncherConfig,
                                       final IntegrationLauncherConfig destinationLauncherConfig,
-                                      final StandardSyncInput syncInput) {
+                                      final StandardSyncInput syncInput,
+                                      @Nullable final String taskQueue) {
```

Comment:
I think it'd be worth adding a comment explaining that this is only nullable so that workflow executions from before the change still work. Could also add a TODO comment that we can make this non-nullable in the future once there aren't any older workflow executions still running